Marlin/language_es.h

```diff
@@ -161,6 +169,12 @@
 #define MSG_ERR_MINTEMP                     _UxGT("Error: Temp Minima")
 #define MSG_ERR_MAXTEMP_BED                 _UxGT("Error: Temp Max Plat")
 #define MSG_ERR_MINTEMP_BED                 _UxGT("Error: Temp Min Plat")
+#define MSG_ERR_Z_HOMING                    _UxGT("G28 Z Prohibido”)
+#define MSG_HALTED                          _UxGT(“IMPRESORA PARADA”)
+#define MSG_PLEASE_RESET                    _UxGT(“Favor de reiniciar“)
```

Comment:
The char '”' is not '"'. This code will cause the string MSG_ERR_Z_HOMING be
```C++
"G28 Z Prohibido”)
#define MSG_HALTED                          _UxGT(“IMPRESORA PARADA”)
#define MSG_PLEASE_RESET                    _UxGT(“Favor de reiniciar“)"
```

Are you sure you can compiled with this "language_es.h" file?